From ef6e8358e99ad4dec57a1cf65f246ca6081dc0d9 Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 09:33:29 -0700
Subject: [PATCH 17/96] HBASE-7113 TestGzipFilter is flaky with jdk1.7

Reason: Test
Author: Jimmy Xiang
Ref: CDH-8954
(cherry picked from commit e15f3c4e08f78ee3816ce3d555d3a73eb5f5cddb)
---
 .../apache/hadoop/hbase/rest/TestGzipFilter.java   |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/test/java/org/apache/hadoop/hbase/rest/TestGzipFilter.java b/src/test/java/org/apache/hadoop/hbase/rest/TestGzipFilter.java
index 570addd..4db967f 100644
--- a/src/test/java/org/apache/hadoop/hbase/rest/TestGzipFilter.java
+++ b/src/test/java/org/apache/hadoop/hbase/rest/TestGzipFilter.java
@@ -116,6 +116,8 @@ public class TestGzipFilter {
     is.read(value, 0, VALUE_1.length);
     assertTrue(Bytes.equals(value, VALUE_1));
     is.close();
+
+    testScannerResultCodes();
   }
 
   @Test
@@ -133,8 +135,7 @@ public class TestGzipFilter {
     assertTrue(contentEncoding == null || !contentEncoding.contains("gzip"));
   }
 
-  @Test
-  public void testScannerResultCodes() throws Exception {
+  void testScannerResultCodes() throws Exception {
     Header[] headers = new Header[3];
     headers[0] = new Header("Content-Type", Constants.MIMETYPE_XML);
     headers[1] = new Header("Accept", Constants.MIMETYPE_JSON);
-- 
1.7.0.4

